.github/workflows/cd.yml

```diff
@@ -39,3 +67,4 @@ jobs:
           name: release
           path: |
             *.vsix
+            package.json
```

Comment:
why upload package.json?

Reply:
we need get vsix's version both pre and stable, and tag with version to repo. 
because the prerelease version will not merge back to remote, so if we need to get the bits version, just upload the package.json